sklearn/feature_selection/tests/test_rfe.py

```diff
@@ -371,6 +376,47 @@ def test_rfe_cv_groups():
     assert est_groups.n_features_ > 0
 
 
+@pytest.mark.parametrize(
+    'importance_getter',
+    [attrgetter('regressor_.coef_'), 'regressor_.coef_'])
+@pytest.mark.parametrize('selector', [RFE, RFECV])
+def test_rfe_wrapped_estimator(importance_getter, selector):
+    # Non-regression test for
+    # https://github.com/scikit-learn/scikit-learn/issues/15312
+    n_features = 10
+    X, y = make_friedman1(n_samples=50, n_features=n_features, random_state=0)
+    estimator = LinearSVR()
+
+    log_estimator = TransformedTargetRegressor(regressor=estimator,
+                                               func=np.log,
+                                               inverse_func=np.exp)
+
+    selector = selector(log_estimator, importance_getter=importance_getter)
+    sel = selector.fit(X, y)
+    assert sel.support_.sum() == n_features // 2
```

Comment:
I don't think this is an issue. Basically the weights are different and we only select 4 features instead of 5 with the other classifier. The optimization method in these classes are different anyway.

Reply:
`n_features_to_select` is kept as `None`, which means the selector has to choose half the number of input features. Thats the reason why I am thinking this as a problem. Am I missing anything?

Reply:
Uhm true. I was thinking it was a criterion based on ranking ratio or something like that. So it seems that there is another bug. I am not sure that this is linked to this PR. I will debug this one to understand.

Reply:
The test is failing for REFCV and not RFE. We used the CV to select the best number of features in this case and not just half. So the code is fine. I think you can pass the number of features expected in the parametrize.

Reply:
Great catch. I will update it then.